tests/test_server_basic.py

```diff
@@ -520,10 +535,11 @@ def test_no_token(dispatcher_live_fixture, async_dispatcher):
 
 
 @pytest.mark.parametrize("selection", ["range", "280200470010.001"])
+@pytest.mark.parametrize("method", ['get', 'post'])
 @pytest.mark.dda
 @pytest.mark.isgri_plugin
 @pytest.mark.xfail
-def test_isgri_image_no_pointings(dispatcher_live_fixture, selection):
+def test_isgri_image_no_pointings(dispatcher_live_fixture, selection, method):
```

Comment:
This test will be moved in the dispatcher-plugin-integral repo.

Reply:
yes!